test/utils.py

```diff
@@ -34,10 +50,17 @@ def assert_tensor_equal(*args, **kwargs):
 
 
 # Asserts that at least `percentage`% of the values are within the absolute tolerance.
-def assert_tensor_close_on_at_least(frame1, frame2, percentage=99.7, abs_tolerance=20):
+def assert_tensor_close_on_at_least(frame1, frame2, percentage=90, abs_tolerance=20):
```

Comment:
Since the default tolerance has changed, did that implicitly changed the tolerance of the existing calls as well? E.g. the ones introduced in https://github.com/pytorch/torchcodec/pull/319/files has a stricter tolerance IIUC - we may need to update those?

Reply:
I am fine using higher tolerances for those too -- as they may have different values for different GPU models anyway